docs/get-invited-and-onboard-to-techpass.md

```diff
@@ -1,139 +1,100 @@
 
-?>- TechPass does not support email accounts which does not have an inbox. For example, LiteMail accounts. If you use such an email account, upgrade it to a standard mailbox before requesting for TechPass.<br>- If you do not see the TechPass onboarding email in your inbox, please check your Junk Email, Deleted Items or Archive folder.<br>- The onboarding email is valid for 30 days. If you do not onboard to TechPass within this 30 days, we will terminate your TechPass account, and you need to sign up again.
+---
 
+## Step 1: Request a TechPass account  
 
-## Step 1: Request for TechPass account
+1. Contact your **project manager** or **reporting officer** to request TechPass and SEED provisioning.  
+2. Provide the following details:  
+   - Organisational email address  
+   - Mobile phone number  
+   - Project name  
 
-1. Contact your project manager or the reporting officer to request for the provisioning of TechPass and SEED, which is optional.
+?> For offshore users not in an ODC, provide:  
+- Offshore reporting officer’s name and email  
+- Offshore company address  
 
-   !> To access services such as SGTS and GCC 2.0 resources through an Internet Device, you need to onboard that device to SEED.
+3. If the invitation is successful, you will receive a **registration email**.
 
-2. Provide the required details in this request such as your organisational email address, mobile phone number and project name.
+---
 
-   Project manager or the reporting officer contacts the sponsoring agency or the tenant admin to invite you to TechPass.
+## Step 2: Complete the registration  
 
-  > **Additional information**:
-  >
-  > **If your request for TechPass provisioning
-  is successful**:
-  >
-  >- A TechPass account is provisioned for you and is in pending state.
-  >- We'll send the TechPass onboarding email with your TechPass account or log in ID. 
-  >- You need to activate the account within 30 days. 
-  >- Your TechPass log in ID's domain is ```techpass.gov.sg```.
-  >- We'll send the initial password by SMS to the registered mobile number.
-  >
-  > **If your request for SEED provisioning is successful**:
-  >
-  >- We'll send the SEED onboarding email immediately.
-  >- This email is valid only for 30 days.
-  >- Ensure that you have activated your TechPass account before proceeding to onboard your Internet Device to SEED.
-  >- If your SEED onboarding email has expired, create a [service request with TechPass](https://go.gov.sg/seed-techpass-support) to receive the SEED onboarding invitation email again. For more information, see [SEED FAQs](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/faqs/seed-faq-general).
+1. Open the **registration link** in your email.  
+2. Fill in the form with the following details:  
+   - Username  
+   - First name and last name  
+   - Mobile number  
+   - Department  
+   - *(Offshore only)* Reporting officer name, email, and company address  
```

Comment:
For offshore users, we classify into odc & non-odc users, only offshore non-odc users need to provide this info. ODC stands for 'Offshore Development Centre'